paperspace/workspace.py

```diff
@@ -24,11 +24,14 @@ def __init__(self, experiments_api, logger=None):
         self.logger = logger or default_logger
 
     @staticmethod
-    def _retrieve_file_paths(dir_name, ignored_files=''):
+    def _retrieve_file_paths(dir_name, ignored_files=None):
         # setup file paths variable
         file_paths = {}
-        ignored_files = ignored_files.split(',')
-        exclude = ['.git', '.idea', '.pytest_cache'] + ignored_files
+
+        exclude = ['.git', '.idea', '.pytest_cache']
+        if ignored_files is not None:
+            exclude += ignored_files.split(',')
```

Comment:
if anything, we could just skip `is not None` part and simply check `if ignored_files:`